pkgdb/src/pkgdb/read.cc

```diff
@@ -393,14 +393,14 @@ PkgDbReadOnly::getPackage( row_id row )
       , 'version',     version
       , 'description', Descriptions.description
       , 'license',     license
-      , 'broken',      iif( ( broken IS NULL )
-                          , json( 'null' )
-                          , iif( broken, json( 'true' ), json( 'false' ) )
-                          )
-      , 'unfree',      iif( ( unfree IS NULL )
-                          , json( 'null' )
-                          , iif( unfree, json( 'true' ), json( 'false' ) )
-                          )
+      , 'broken',      CASE WHEN broken IS NULL THEN json( 'null' )
```

Comment:
Question: is there a reason the capitalization of `json` is different on this line and the next?

Reply:
That was a typo, I've corrected it.
It's case insensitive so either works; but you're right that they were inconsistent and wonky.